### PR TITLE
fix: remove ansible commands from config template.

### PR DESCRIPTION
**Pull request by archnode**

### CODE REVIEW

File: templates/config.j2 [hunk 1]
```diff
@@ -58,15 +58,11 @@ providers:
 {% endif %}
     defaultRule: {{ traefik_docker_default_rule }}
 
-  ansible.builtin.file:
```
Comment (tboerger) [inline templates/config.j2:61]: file would be right here, just ansible.builtin. got to be removed :)

File: templates/config.j2 [hunk 2]
```diff
@@ -58,15 +58,11 @@ providers:
...
-    filename: /etc/traefik/static.yml
-    watch: true
 {% if traefik_api_enabled %}
 
 api:
   insecure: {{ traefik_api_insecure | lower }}
   dashboard: {{ traefik_api_dashboard | lower }}
-  ansible.builtin.debug: {{ traefik_api_debug | lower }}
```
Comment (tboerger) [inline templates/config.j2:69]: Same here

**Review by tboerger — COMMENTED** [review]
The sections itself are right, just the keys have been replaced wrong. Thanks for the effort.

File: templates/config.j2 (hunk 1 above)
Comment (archnode) [reply templates/config.j2:61]: Gotcha :sweat_smile:

**Pull request merged**